.github/workflows/build.yml

```diff
@@ -76,6 +77,12 @@ jobs:
       - name: Validate | Fmt
         run: gofmt -l cmd/lukso
```

Comment:
this step would be checked in the Golangci-lint below so maybe if golangci stays below we can remove this step